server/routes/reviews.js

```diff
@@ -0,0 +1,73 @@
+const db = require('APP/db/models');
+const Review = db.Review;
+
+const Router = require('express').Router();
+
+// GET /api/reviews/ - gets all the review
+Router.get('/', (req, res, next) => {
+	Review.findAll({})
+	.then((foundReviews) => {
+		res.json(foundReviews);
+	})
+	.catch(next);
+});
+
+//GET /api/reviews/:reviewId - gets a single review
+Router.get('/:reviewId', (req, res, next) => {
+	Review.findById(
+		req.params.reviewId
+	)
+	.then((singleReview) => {
+		res.json(singleReview);
+	})
+	.catch(next);
+});
+
```

Comment:
I think we should add at least 2 more GET routes for Review: 
1. '/api/reviews/:productId' - get all the reviews for one product
2. '/api/reviews/user/:userId' - get all the reviews written by one user (or author)
For these, we can probably use the get methods made through the associations we set up earlier right?